Scoop/Controller/Onboarding/ProfilePictureViewController.swift

```diff
@@ -90,11 +90,12 @@ class ProfilePictureViewController: OnboardingViewController {
                 self.dismiss(animated: true)
                 self.containerDelegate?.dismiss(animated: true)
             case .failure(let error):
-                print("Request Failed")
                 print(error.localizedDescription)
```

Comment:
A little cleaner imo: `print("Request Failed: \(error.localizedDescription)")`
If you have a lot of error handling through the codebase, it may help to either 1) make the print statement a little more descriptive or 2) create custom errors